client/my-sites/plans-features-main/plan-type-selector.tsx

```diff
@@ -152,6 +156,7 @@ export const IntervalTypeToggle: React.FunctionComponent< IntervalTypeProps > =
 					path={ generatePath( props, {
 						intervalType: 'monthly',
 						get_domain: domainFromHomeUpsellFlow,
+						...redirectTo,
```

Comment:
Is this really needed? Looks like the `SegmentedControl.Item` component does nothing with the `redirect_to` prop.

Reply:
I'm not sure I follow. The `redirectTo` property is added to the `generatePath` function that generates a path. We need this so that we pass `redirect_to` parameter between the `Yearly` and `Monthly` toggles.

Otherwise,  when the user switches between plan types, it'll lose the `redirect_to` parameter.

Reply:
> The `redirectTo` property is added to the `generatePath` function that generates a path

Oh, I completely missed that, sorry. I thought it was a property for the `SegmentedControl.Item` component.